src/modalities/loss_functions.py

```diff
@@ -24,11 +24,11 @@ def __call__(self, forward_batch: InferenceResultBatch) -> torch.Tensor:
 
 
 class CLMCrossEntropyLoss(Loss):
-    def __init__(self, target_key: str, prediction_key: str, tag: str = "CLMCrossEntropyLoss"):
+    def __init__(self, target_key: str, prediction_key: str, tag: str = "CLMCrossEntropyLoss", reduction: str = "sum"):
```

Comment:
I think, we should set this reduction to mean. Even, if that means that the loss aggregation for logging purposes gets a bit wonky.
Suddenly switching to sum here will have unforeseen effects on model training. This is probably equivalent to multiplying the learning rate with batch_size * max_sequence_length.